README.md

```diff
@@ -19,6 +19,7 @@ To contribute:
 
 | Name                                                                                                                                                       | Location                                | Application Period    | Notes                                                                                          | Date Posted |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- | ----------- |
+| [Nestle](https://jobdetails.nestle.com/job/North-York-Web-Development-Intern-ON/981648201/?feedId=256801&utm_source=NestleCareers)                         | Toronto                                 | Before October 9      | Web Development Intern                                                                         | Unknown     |
```

Comment:
Could you add the date as today's date instead of unknown?

Reply:
Sure

Reply:
Thank you thank you

Reply:
No problem!